muonShieldOptimization/run_fixedTarget.py

```diff
@@ -256,79 +273,79 @@ def init():
 ctime = timer.CpuTime()
 print(' ')
 print("Macro finished successfully.")
-print("Output file is ",  outFile)
-print("Real time ",rtime, " s, CPU time ",ctime,"s")
+print(f"Output file is {outFile}")
+print(f"Real time {rtime} s, CPU time {ctime} s")
 # ---post processing--- remove empty events --- save histograms
-tmpFile = outFile+"tmp"
-if ROOT.gROOT.GetListOfFiles().GetEntries()>0:
- fin   = ROOT.gROOT.GetListOfFiles()[0]
+tmpFile = outFile + "tmp"
+if ROOT.gROOT.GetListOfFiles().GetEntries() > 0:
+    fin = ROOT.gROOT.GetListOfFiles()[0]
 else:
- fin = ROOT.TFile.Open(outFile)
+    fin = ROOT.TFile.Open(outFile)
 fHeader = fin.FileHeader
```

Comment:
There is a deprecation warning:
```
 DeprecationWarning: The attribute syntax for TFile is deprecated and will be removed in ROOT 6.34. Please use TFile["FileHeader"] instead of TFile.FileHeader
  fHeader = fin.FileHeader

```

Reply:
As of #788 , this is also something already available in FairShip, even though we still use 6.30 by default.